### PR TITLE
Added wNEAR/USN Pool

### PR DESCRIPTION
**Pull request by specialist0x**
image

### CODE REVIEW

File: src/state/stake/stake-constants.ts [hunk 1]
```diff
@@ -535,6 +536,14 @@ const AURORA_POOLS: StakingTri[] = [
     stableSwapPoolName: StableSwapPoolName.NUSD_USDC_USDT,
     friendlyFarmName: STABLESWAP_POOLS[StableSwapPoolName.NUSD_USDC_USDT].friendlyName,
     isFeatured: true
+  }),
+  createMCV2Pool({
+    ID: 40,
+    poolId: 33,
+    tokens: [WNEAR[ChainId.AURORA], USN[ChainId.AURORA]],
+    lpAddress: '0xA36DF7c571bEbA7B3fB89F25dFc990EAC75F525A',
+    rewarderAddress: ZERO_ADDRESS,
```
Comment (memacius32) [inline src/state/stake/stake-constants.ts:545]:
```
rewarderAddress: ZERO_ADDRESS,
```
I think this isn't needed

**Review by memacius32 — APPROVED** [review]
Approving with one comment

**Comment by memacius32** [comment]
Approving with one comment

**Comment by specialist0x** [comment]
@0xchain1 @0xdon This is ready to go, feel free to merge in when ready

**Pull request merged**